Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/Views/AssetBrowserTreeView.cpp

```diff
@@ -181,6 +181,18 @@ namespace AzToolsFramework
             CaptureTreeViewSnapshot();
         }
 
+        void AssetBrowserTreeView::dropEvent([[maybe_unused]] QDropEvent* event)
```

Comment:
Nit: `event` is used below so can get rid of the `[[maybe_unused]]`

Reply:
Oops!